backend/model_server/encoders.py

```diff
@@ -190,7 +190,9 @@ def get_embedding_model(
 
     if model_name not in _GLOBAL_MODELS_DICT:
         logger.info(f"Loading {model_name}")
-        model = SentenceTransformer(model_name)
+        model = SentenceTransformer(
+            model_name_or_path=model_name, trust_remote_code=True
+        )
```

Comment:
This is related to the needing to also install nomic-bert-2048
there is a script that has to be executed to use the model (unsure when) that is located in nomic-bert-2048 and not in nomic-embed-text-v1 (a couple .py scripts you can see [here](https://huggingface.co/nomic-ai/nomic-bert-2048/tree/main))

Not 100% sure though